WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -467,38 +467,37 @@ private extension MainTabBarController {
     }
 }
 
-// MARK: - Reviews Tab Badge Updates
+// MARK: - Hub Menu Tab Badge Updates
 //
 private extension MainTabBarController {
 
     /// Setup: KVO Hooks.
     ///
-    func startListeningToReviewsTabBadgeUpdates() {
+    func startListeningToHubMenuTabBadgeUpdates() {
         NotificationCenter.default.addObserver(self,
-                                               selector: #selector(loadReviewsTabNotificationCountAndUpdateBadge),
+                                               selector: #selector(loadHubMenuTabNotificationCountAndUpdateBadge),
                                                name: .reviewsBadgeReloadRequired,
                                                object: nil)
     }
 
-    @objc func loadReviewsTabNotificationCountAndUpdateBadge() {
+    @objc func loadHubMenuTabNotificationCountAndUpdateBadge() {
         guard let siteID = stores.sessionManager.defaultStoreID else {
             return
         }
 
         let action = NotificationCountAction.load(siteID: siteID, type: .kind(.comment)) { [weak self] count in
-            self?.updateReviewsTabBadge(count: count)
+            self?.updateHubMenuTabBadge(count: count)
         }
         stores.dispatch(action)
     }
 
-    /// Displays or Hides the Dot on the Reviews tab, depending on the notification count
+    /// Displays or Hides the Dot on the Hub Menu tab, depending on the notification count
     ///
-    func updateReviewsTabBadge(count: Int) {
-        //TODO-5509: handle reviews badge
-        guard !isHubMenuFeatureFlagOn else {
```

Comment:
❓ do we want to keep the feature flag disabled logic just in case we aren't ready to release the Home Screen updates in release 8.4 (like due to screenshots or other reasons like aligning with Android)?

Reply:
Yes it makes sense. I updated the logic here 35d3524716aecb2392c4df5404c7994c4deec33a